packages/appstore/src/components/modals/real-wallets-upgrade/components/title-text/title-text.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Text } from '@deriv/components';
+import './title-text.scss';
+
+const TitleText = ({ children, className }: React.PropsWithChildren<{ className?: string }>) => {
+    return (
+        <Text className={classNames('title-text', className)} color='prominent' align='center'>
```

Comment:
@aum-deriv like others I believe we can handle this by using `<Text/>` but if you want to have it as a separate component can we also pass `color` and `align` as props. maybe in the future we have the same design with different alignment or bgColor. then we can use this one there too.

if you agree please make a type for the probs instead of having it inline as well 🙏